boards/samd20_xplained_pro/.cargo/config

```diff
@@ -0,0 +1,18 @@
+# samd21 is a Cortex-M0 and thus thumbv6m
+
+[build]
+target = "thumbv6m-none-eabi"
+
+[target.thumbv6m-none-eabi]
+runner = 'arm-none-eabi-gdb -q -x openocd.gdb'
+#runner = 'probe-run --chip ATSAMD21G18A'
+
+rustflags = [
+
+  # This is needed if your flash or ram addresses are not aligned to 0x10000 in memory.x
+  # See https://github.com/rust-embedded/cortex-m-quickstart/pull/95
+  # "-C", "link-arg=--nmagic",
+
+  "-C", "linker=lld",
```

Comment:
Don't include this, and uncomment the nmagic line above to align with the rest of the configs. Also nit pick, can you name it config.toml? you don't need to leave off the .toml anymore.

Reply:
Removed and renamed :)